staff/pedro-garriganogues/cv/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE <!DOCTYPE html>
+<html>
+<head>
+
+    <h1>Pedro Garriga-Nogués</h1>
+    <hr>
+    <dl>
+        <img src="handsome.jpg" alt="Oooopssss">
+        <dt><a href="https://github.com/PedroGarrigaNogues">GitHub </a><a href="https://www.linkedin.com/in/pedro-garriga-nogu%C3%A9s-2ab577a5/">Linkedin</a></dt>
+        <dt>Email:</dt>
+        <dt>pedrogarriganogues@outlook.com</dt>
```

Comment:
If you decide to use dl dt and dd tags, you should use some dd dor the explanations of each term. check the info here: https://www.w3schools.com/tags/tag_dt.asp